Adyen/UI/UI Style/CoreColors.swift

```diff
@@ -10,65 +10,78 @@ import UIKit
 extension UIColor {
     
     /// :nodoc:
-    internal enum AdyenCore {
```

Comment:
I believe we called this `AdyenCore` because there was some collisions on Pods?

Reply:
no I recall we called it `AdyenCore` because in DropIn there was another extension to UIColor which has some duplicates and we called it `AdyenDropIn`, now there is only one extension in `Adyen` target with public modifiers to avoid code duplication and removed the extension in DropIn.